File: rust/fizzy.rs
use std;

fn fizzbuzz(n: int) -> str {
	let mod3 = n % 3 == 0;
	let mod5 = n % 5 == 0;

	if (mod3 && mod5) {
		ret "FizzBuzz";
	}
	else if (mod3) {
		ret "Fizz";
	}
	else if (mod5) {
		ret "Buzz";
	}
	else {
		ret core::int::to_str(n, 10u);
	}
}

fn main() {
	core::int::range(1, 101, { |n| 
		std::io::println(fizzbuzz(n));
	});
}

File: python/fizzy.py
#!/usr/bin/env python

# Backwards compatibility with Python 2;
# Python 3 allows print() in lambdas.
from __future__ import print_function

import multiprocessing

def fizzbuzz(n):
    x, y = n % 3, n % 5

    if (x, y) == (0, 0):
        return "FizzBuzz"
    elif x == 0:
        return "Fizz"
    elif y == 0:
        return "Buzz"
    else:
        return str(n)

def main():
    p = multiprocessing.Pool()

    map(
        (lambda s: print(s)),
        p.map(
            fizzbuzz,
            range(1, 100)
        )
    )

if __name__=="__main__":
    main()
